src/Modules/UngradedModules.jl

```diff
@@ -1976,7 +1985,7 @@ function sub(M::SubQuo, V::Vector{<:SubQuoElem}, task::Symbol = :none, check = t
   if check
     @assert all(x -> x.parent === M, V)
   end
-  t = SubQuo(M.F, [x.repres for x in V])
+  t = SubQuo(M.F, Vector{FreeModElem}([repres(x) for x in V]))
```

Comment:
This would be slightly more idiomatic, I think:
```suggestion
  t = SubQuo(M.F, FreeModElem[repres(x) for x in V])
```